README.md

```diff
@@ -1,115 +1,157 @@
+This library is a standalone library designed for all java environments. The library acts as a client for the Kinvey REST API and can be used for building Android apps and Java6 server applications.
 
-It is recommended you use either IntelliJ or Android Studio. Eclipse is NOT recommended.
+It is recommended to use either IntelliJ or Android Studio. Eclipse is NOT recommended.
 
-##Documentation
-Refer http://devcenter.kinvey.com/android for complete documentation of the library APIs and usage.
+## Documentation
+Refer to https://devcenter.kinvey.com/android-v2 for complete documentation of the library APIs and usage.
 
-##Overview of the Library -
+## Overview of the Library
 
 The codebase is made of the following key projects at the top level (under java-library): 
 
-###java-api-core 
+### java-api-core 
 The core of the library. Most of the library logic is written here. This project contains most of the underlying networking, user management, caching logic. Things that are platform specific (android-specific or standalone-java-specific) are represented as interfaces / abstract classes, and implemented in the other libraries described below.
 
-###android-lib
+### android-lib
 The wrapper library for android, built on top of java-api-core. All the android specific implementation goes here. Most of the classes in this library extend from the ones in java-api-core.
 
-###java-lib
+### java-lib
 The wrapper library for java, built on top of java-api-core. All the standalone-java specific implementation goes here. Most of the classes in this library extend from the ones in java-api-core.
 
 ###android-secure
 Encryption module built on top of android-lib. Rarely used; not compiled into the standard build process. This may be requested by certain customers who need encryption in their app.
 
-###samples 
-Samples built on top of the libraries. This is a submodule, the full source for samples is under https://github.com/KinveyApps
+### samples 
+Samples built on top of the libraries. This is a submodule, the full source for samples is under https://github.com/Kinvey/java-library/tree/ver2.x/samples
 
 ## Build
-Pre-requisites:
 
-* [android sdk](http://developer.android.com/sdk/index.html)
-* [gradle build system](http://gradle.org/)
-
-```
-gradle clean build
-```
-
-```
-gradle release
-```
-
-```
-gradle test jacocoTestReport
-```
+Prerequisites:
 
-
-##Legacy Build (DEPRECATED!)
-### Regenerate Javadocs
+* [Android SDK](http://developer.android.com/sdk/index.html)
+* Set JAVA_HOME
+* Set ANDROID_HOME
+* Download android_sdk/platforms/android-19, android_sdk/platforms/android-10
 
 ```
-rm -r <devcenter.home>/content/reference/android/api/*
-cd <project.home> 
-mvn -Pdev javadoc:javadoc install
+./gradlew clean release
 ```
 
-### Release
+The command should generate a .zip archive in `/release/zipped` containing .aar and .jar files.
 
+## How to enable TLSv1.1/TLSv1.2 on Android v4.1-v4.4.4
```

Comment:
@vinaygahlawat Should we follow suit and downplay v1.1 as in the other documents? (This should also require tweaking the example code a bit to not enable this version.)

Reply:
@boevski I think in the case of the example that it is still ok to discuss 1.1 and 1.2, as this is just enabling these protocol versions, which is correct. As you mentioned before, we can still emphasize 1.2 in our instructions, as long as we still allow for both 1.1 and 1.2.